docs/AddNewOp.md

```diff
@@ -1,58 +1,51 @@
-quality operator specification to expand our operator set.
-
-We are maintaining [a list](https://github.com/onnx/onnx/issues/1646) in our
-GitHub issue. Please check it and post your request following this thread.
-
-## Step 1: Determine Whether We Should Add the Operator
-Determine whether if the proposed operator is common enough, usually it
-should/will be supported by at least two frameworks, such as PyTorch/Caffe2,
-Tensorflow, MxNet, etc.
-
-## Step 2: Determine the Operator Should Be Added as Operator or Function
-If the requested operator can be easily expressed as several (e.g., 2 or 3)
-existing operators, we can define it as a Function (i.e., composite operator).
-MeanVarianceNormalization is the first example of registering an operator
-as a Function.
-
-## Step 3: Define and polish the spec of the proposed operator.
-The spec should include:
-1. opset version of the added operator. Check out our
+# Proposing and submitting a new operator or function to ONNX
+
+Operators are the basic building blocks that define ONNX model. With a rich set of operators, ONNX can describe most DNN and ML models from various frameworks. Improving the process of accepting and adding a new operator to ONNX is crucial for ONNX standard future and its viability.
+
+In this document, we describe the process of accepting a new proposed operator and how to properly submit a new operator as part of ONNX standard. The goal is to improve on what we currently have based on our experience, learning and feedbacks we gathered from the community. 
+
+Adding new operator to ONNX is vital to keep up with latest model architectures and to keep ONNX standard relevant.
+
+# Accepting new operator
+The goal of ONNX standard isn't to implement every possible existing operator from all available frameworks. The goal is to cover most models and keep evolving ONNX to cover future models.
+
+## Proposing a new operator
+In order to propose a new operator, the following is needed:
+1. If the operator can be composed by other ONNX operators, then it should be a function and not an operator (we have a function in ONNX : MeanVarianceNormalization).
+2. If the operators can be split to new primitives, propose those primitives instead and make the operator a function.
+3. Based on a model. This will help us understand the usage and that it solves an actual problem. For the case of the model being private or IP and can't be shared, the operator doesn't belong to the standard and should be implemented as custom OP.
+4. The operator needs to be implemented by at-least one (well-known) framework. This help us to understand the actual behavior of the operator and its usage.
+5. Operator signature and behavior:
+    1. If the operator is available in numpy, prefer numpy semantics.
+    2. If the operator is available in more than one frameworks, make sure that your design is general and cover those frameworks.
+6. Prefer attributes over inputs.
```

Comment:
Is it `prefer inputs over attributes`? I am fine with both.

Reply:
It is based on the SIG feedback, attributes are easy to optimize.

Reply:
Sounds good.